src/systems/space/spawn_space.rs

```diff
@@ -7,33 +7,28 @@ use bevy::{
 use rand::random;
 
 pub fn spawn_random_empty_space_background(mut commands: Commands, asset_server: Res<AssetServer>) {
-    let space_tile_size = 1920.0;
-    let total_area_around_player = 1920.0 * 1.5;
-    let half_space_tile_size = 1920.0 * 0.5;
-
-    let rand_texture = random::<SpaceSprite>();
+    let tile_size = 1920.0;
+    let random_texture = random::<SpaceSprite>();
 
     for x in 0..5 {
         for y in 0..5 {
-            let space_grid_position = Vec2::new(x as f32, y as f32);
+            let grid_position = Vec2::new(x as f32, y as f32);
 
             let space = Space {
                 asset: random(),
                 size: Vec2 {
-                    x: space_tile_size,
-                    y: space_tile_size,
+                    x: tile_size,
+                    y: tile_size,
                 },
-                space_grid_position,
-                space_location: Vec3::new(
-                    (space_grid_position.x * space_tile_size) - total_area_around_player
```

Comment:
Which is how it was being used here :face_with_head_bandage:

No blame... but we're idiots!